src/main/resources/data/computercraft/lua/rom/apis/colors.lua

```diff
@@ -332,3 +332,14 @@ function rgb8(r, g, b)
         return packRGB(r, g, b)
     end
 end
+
+--- Converts the given color to a paint/blit hex character (0-9a-f).
+--
+-- This is equivalent to converting floor(log_2(color)) to hexadecimal.
+--
+-- @tparam number color The color to convert.
+-- @treturn string The blit hex code of the color.
+function toBlit(color)
+    expect(1, color, "number")
+    return string.format("%x", math.floor(math.log(color) / math.log(2)))
```

Comment:
There is a bit of me which is tempted to do what I've done elsewhere. First declare a look up table (like window.lua) and then do `colour_lookup[color] or string.format(...)`. Not sure if it's worth it, but (should) be much faster in the common case. I should probably benchmark this first.

Reply:
Probably better to fall back to this, yeah. Will just copy the window table and leave window.lua untouched for the time being (since it's a bit more optimised)